stellargraph/mapper/full_batch_generators.py

```diff
@@ -519,7 +520,10 @@ def flow(self, node_ids, targets=None):
             if len(targets) != len(node_ids):
                 raise TypeError("Targets must be the same length as node_ids")
 
-        node_indices = self.graph.node_ids_to_ilocs(node_ids)
+        if use_ilocs == True:
```

Comment:
`use_ilocs` is already a boolean, so a comparison is not necessary: `x == True` is equivalent to `x` (and `x == False` is equivalent to `not x`):

```suggestion
        if use_ilocs:
```

(There's a bit more discussion of this at https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html )